### ISSUE TITLE
Page titles should be unique and descriptive

### ISSUE THREAD

**Issue by briri** [DMPRoadmap/roadmap#1441]
The codebase current pulls the application name out of the `branding.yml` and uses it as the page title in `application.rb`. Each of these titles should be unique though across the site.  

This seems like the accepted approach to handling this: https://rubyplus.com/articles/3321-Pretty-Page-Title-in-Rails-5

The page title should match the content of the `<h1>` of the page.

Also, update the home page `<h1>` so that it is more descriptive (e.g. 'Welcome to DMPRoadmap' instead of 'Welcome')

This feedback was provided from the UCOP electronic accessibility review. It was defined as having 'High' priority.

**Comment by stephaniesimms** [comment]
can't test UI visually. closing issue.
